exampleProject/build.gradle.kts

```diff
@@ -0,0 +1,62 @@
+apply(plugin = "java")
+apply(plugin = "com.google.protobuf")
+apply(plugin = "idea")
+
+repositories {
+    maven("https://plugins.gradle.org/m2/" )
+}
+
+buildscript {
+    repositories {
+        maven("https://plugins.gradle.org/m2/")
+    }
+    dependencies {
+        classpath("com.google.protobuf:protobuf-gradle-plugin:0.8.7-SNAPSHOT")
+    }
+}
+
+dependencies {
+    compile("com.google.protobuf:protobuf-java:3.0.0")
+    compile("io.grpc:grpc-stub:1.0.0-pre2")
+    compile("io.grpc:grpc-protobuf:1.0.0-pre2")
+    if (JavaVersion.current().isJava9Compatible) {
+        // Workaround for @javax.annotation.Generated
+        // see: https://github.com/grpc/grpc-java/issues/3633
+        compile("javax.annotation:javax.annotation-api:1.3.1")
+    }
+    // Extra proto source files besides the ones residing under
+    // "src/main".
+    "protobuf"(files("lib/protos.tar.gz"))
+    "protobuf"(fileTree("ext/"))
```

Comment:
using string configuration access but i should probably look into adding an extension scoped to the dependency handler